### DISCUSSION TITLE
Provide More Convenient Installation For Dolphinscheduler Cluster

### DISCUSSION THREAD

**Discussion by zzzhangqi** [apache/dolphinscheduler#10591] [General]
Hi ~

Most people who use dolphinscheduler are developers.But they are not familiar with docker and kubernetes.Including my colleagues and friends.This makes it more difficult for them to deploy dolphinscheduler cluster.

so,I know that open source paas [Rainbond](https://github.com/goodrain/rainbond) supports one-click deployment of applications,users install and use dolphinscheduler cluster through the rainbond app store.

I have seen similar documents in other open source communities.
- [Apollo](https://www.apolloconfig.com/#/zh/deployment/third-party-tool-rainbond)
- [Nacos](https://nacos.io/zh-cn/blog/use-nacos-with-rainbond.html)

so,I hope to contribute this kind of installation documentation to the dolphinscheduler community,It will be more convenient for users to install dolphinscheduler clusters.

Where can I submit the document to PR if possible

**Comment by ruanwenjun** [comment]
LGTM, welcome to contribute.

**Comment by zzzhangqi** [comment]
Thank you for your reply.Please where should I submit it ? Wesite blog or Website install doc ?

**Reply by ruanwenjun** [reply]
I think you can submit PR to https://github.com/apache/dolphinscheduler-website.

And before you begin to work, you can submit your plan to dev@dolphinscheduler.apache.org.

**Reply by zzzhangqi** [reply]
> I think you can submit PR to https://github.com/apache/dolphinscheduler-website.
> 
> And before you begin to work, you can submit your plan to [dev@dolphinscheduler.apache.org](mailto:dev@dolphinscheduler.apache.org).

no problem
